app/__init__.py

```diff
@@ -14,6 +15,18 @@
 login_manager = LoginManager()
 login_manager.session_protection = 'strong'
 login_manager.login_view = 'auth.login'
+oauth = OAuth()
+github = oauth.remote_app(
```

Comment:
Also, it makes more sense for me to have this whole thing go into config.py and then import it into __init__ since that is where all of the other configuration stuff is currently kept.